__tests__/app.test.js

```diff
@@ -22,3 +22,27 @@ describe("Get Categories", () => {
 			});
 	});
 });
+
+describe("Get Reviews", () => {
+	test("Return reviews and a status code 200", () => {
+		return request(app)
+			.get("/api/reviews")
+			.expect(200)
+			.then((result) => {
+				const expected = {
+					title: "Agricola",
```

Comment:
We don't wanna check specific data here in case it changes. Instead, we can check the properites and data types of the values :)